<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Nuovo paziente</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="userLabel">
     <property name="text">
      <string>Segretario - Nuovo paziente</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="nomeLabel">
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="cognomeLabel">
     <property name="text">
      <string>Cognome:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="codiceLabel">
     <property name="text">
      <string>Codice fiscale:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="residenzaLabel">
     <property name="text">
      <string>Residenza:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="viaEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Via</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="numCivEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="placeholderText">
      <string>Numero Civico</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLineEdit" name="cittaEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="placeholderText">
      <string>Citt√†</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLineEdit" name="regioneEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="placeholderText">
      <string>Regione</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="cellulareLabel">
     <property name="text">
      <string>Cellulare:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <layout class="QHBoxLayout" name="indLogLayout">
     <item>
      <widget class="QPushButton" name="indietroButton">
       <property name="text">
        <string>INDIETRO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="logoutButton">
       <property name="text">
        <string>LOGOUT</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="QLineEdit" name="nomeEdit">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Nome</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="4">
    <widget class="QLineEdit" name="cognomeEdit">
     <property name="placeholderText">
      <string>Cognome</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="4">
    <widget class="QLineEdit" name="codiceEdit">
     <property name="placeholderText">
      <string>Codice fiscale</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="4">
    <widget class="QLineEdit" name="cellulareEdit">
     <property name="placeholderText">
      <string>Cellulare</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="5">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QScrollArea" name="preScroll">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>874</width>
          <height>254</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Prescrizioni mediche registrate</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <layout class="QFormLayout" name="prescrizioniLayout"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0" colspan="5">
    <widget class="QLineEdit" name="prescrEdit">
     <property name="placeholderText">
      <string>Prescrizione</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="5">
    <widget class="QPushButton" name="aggPrescrButton">
     <property name="text">
      <string>Aggiungi prescrizione medica</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="5">
    <widget class="QPushButton" name="salvaButton">
     <property name="text">
      <string>Salva</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <layout class="QVBoxLayout" name="dettLayout">
     <item>
      <widget class="QLabel" name="dettLabel">
       <property name="text">
        <string>Dettagli paziente</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
